pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -114,6 +136,7 @@ func (ds *dockerService) RunPodSandbox(config *runtimeapi.PodSandboxConfig) (str
 
 	// Do not invoke network plugins if in hostNetwork mode.
 	if nsOptions := config.GetLinux().GetSecurityContext().GetNamespaceOptions(); nsOptions != nil && nsOptions.HostNetwork {
+		ds.setNetworkReady(createResp.ID, true)
```

Comment:
@freehan Done!  Can you double-check my usage of named return variables and defer and pointers?